atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,226 @@
+            eps=eps,
+            min_samples=min_samples,
+            metric=SimilarityMetric.cosine.value,
+            min_cluster_size=min_cluster_size
+        )
+
+        # Expose clusters and similarity matrix as instance variables
+        for index in range(len(medoids)):
+            cluster = {
+                "cluster_id": str(index)
+                }
+            self.weaviate_client.add_embeddings(CollectionNames.CLUSTERCENTER.value, medoids[index].tolist(), cluster)
```

Ruff (0.11.9)

Line too long (118 > 88)

(E501)